en/deploy/README.md

```diff
@@ -130,159 +130,118 @@ Your code is now on GitHub. Go and check it out!  You'll find it's in fine compa
-(mvenv) $ python manage.py createsuperuser
+$ workon $USER.pythonanywhere.com
+(yourusername.pythonanywhere.com) $ cd $USER.pythonanywhere.com
+(yourusername.pythonanywhere.com) $ python manage.py createsuperuser
 ```
 
-## Publishing our blog as a web app
-
-Now our code is on PythonAnywhere, our virtualenv is ready, and the database is initialized. We're ready to publish it as a web app!
-
-Click back to the PythonAnywhere dashboard by clicking on its logo, and then click on the **Web** tab. Finally, hit **Add a new web app**.
-
-After confirming your domain name, choose **manual configuration** (N.B. – *not* the "Django" option) in the dialog. Next choose **Python 3.6**, and click Next to finish the wizard.
-
-> **Note** Make sure you choose the "Manual configuration" option, not the "Django" one. We're too cool for the default PythonAnywhere Django setup. ;-)
-
+If you like you can also take a look at your code on PythonAnywhere by typing `tree`:
 
-### Setting the virtualenv
-
-You'll be taken to the PythonAnywhere config screen for your webapp, which is where you'll need to go whenever you want to make changes to the app on the server.
-
-<img src="images/pythonanywhere_web_tab_virtualenv.png" />
-
-In the "Virtualenv" section, click the red text that says "Enter the path to a virtualenv", and enter `/home/<your-PythonAnywhere-username>/my-first-blog/myvenv/`. Click the blue box with the checkmark to save the path before moving on.
-
-> **Note** Substitute your own PythonAnywhere username as appropriate. If you make a mistake, PythonAnywhere will show you a little warning.
-
-
-### Configuring the WSGI file
-
-Django works using the "WSGI protocol", a standard for serving websites using Python, which PythonAnywhere supports. The way we configure PythonAnywhere to recognize our Django blog is by editing a WSGI configuration file.
-
-Click on the "WSGI configuration file" link (in the "Code" section near the top of the page – it'll be named something like `/var/www/<your-PythonAnywhere-username>_pythonanywhere_com_wsgi.py`), and you'll be taken to an editor.
-
-Delete all the contents and replace them with the following:
-
-{% filename %}&lt;your-username&gt;_pythonanywhere_com_wsgi.py{% endfilename %}
-```python
-import os
-import sys
-
-path = os.path.expanduser('~/my-first-blog')
-if path not in sys.path:
-    sys.path.append(path)
-
-os.environ['DJANGO_SETTINGS_MODULE'] = 'mysite.settings'
-
-from django.core.wsgi import get_wsgi_application
-from django.contrib.staticfiles.handlers import StaticFilesHandler
-application = StaticFilesHandler(get_wsgi_application())
+{% filename %}PythonAnywhere command-line{% endfilename %}
+```
+$ tree
+.
+├── blog
+│   ├── __init__.py
+│   ├── admin.py
+│   ├── migrations
+│   │   ├── 0001_initial.py
+│   │   └── __init__.py
+│   ├── models.py
+│   ├── tests.py
+│   └── views.py
+├── manage.py
+└── mysite
+    ├── __init__.py
+    ├── settings.py
+    ├── urls.py
+    └── wsgi.py
 ```
```

Comment:
The output of `tree` is now much larger than shown in the tutorial:
<details><summary>
18 directories, 84 files (click to expand)
</summary>
<p>

```
.
├── README.txt
└── dasg3.pythonanywhere.com
    ├── blog
    │   ├── __init__.py
    │   ├── __pycache__
    │   │   ├── __init__.cpython-36.pyc
    │   │   ├── admin.cpython-36.pyc
    │   │   └── models.cpython-36.pyc
    │   ├── admin.py
    │   ├── apps.py
    │   ├── migrations
    │   │   ├── 0001_initial.py
    │   │   ├── __init__.py
    │   │   └── __pycache__
    │   │       ├── 0001_initial.cpython-36.pyc
    │   │       └── __init__.cpython-36.pyc
    │   ├── models.py
    │   ├── tests.py
    │   └── views.py
    ├── db.sqlite3
    ├── manage.py
    ├── mysite
    │   ├── __init__.py
    │   ├── __pycache__
    │   │   ├── __init__.cpython-36.pyc
    │   │   ├── settings.cpython-36.pyc
    │   │   └── urls.cpython-36.pyc
    │   ├── settings.py
    │   ├── urls.py
    │   └── wsgi.py
    └── static
        └── admin
            ├── css
            │   ├── base.css
            │   ├── changelists.css
            │   ├── dashboard.css
            │   ├── fonts.css
            │   ├── forms.css
            │   ├── login.css
            │   ├── rtl.css
            │   └── widgets.css
            ├── fonts
            │   ├── LICENSE.txt
            │   ├── README.txt
            │   ├── Roboto-Bold-webfont.woff
            │   ├── Roboto-Light-webfont.woff
            │   └── Roboto-Regular-webfont.woff
            ├── img
            │   ├── LICENSE
            │   ├── README.txt
            │   ├── calendar-icons.svg
            │   ├── gis
            │   │   ├── move_vertex_off.svg
            │   │   └── move_vertex_on.svg
            │   ├── icon-addlink.svg
            │   ├── icon-alert.svg
            │   ├── icon-calendar.svg
            │   ├── icon-changelink.svg
            │   ├── icon-clock.svg
            │   ├── icon-deletelink.svg
            │   ├── icon-no.svg
            │   ├── icon-unknown-alt.svg
            │   ├── icon-unknown.svg
            │   ├── icon-yes.svg
            │   ├── inline-delete.svg
            │   ├── search.svg
            │   ├── selector-icons.svg
            │   ├── sorting-icons.svg
            │   ├── tooltag-add.svg
            │   └── tooltag-arrowright.svg
            └── js
                ├── SelectBox.js
                ├── SelectFilter2.js
                ├── actions.js
                ├── actions.min.js
                ├── admin
                │   ├── DateTimeShortcuts.js
                │   └── RelatedObjectLookups.js
                ├── calendar.js
                ├── cancel.js
                ├── change_form.js
                ├── collapse.js
                ├── collapse.min.js
                ├── core.js
                ├── inlines.js
                ├── inlines.min.js
                ├── jquery.init.js
                ├── popup_response.js
                ├── prepopulate.js
                ├── prepopulate.min.js
                ├── prepopulate_init.js
                ├── timeparse.js
                ├── urlify.js
                └── vendor
                    ├── jquery
                    │   ├── LICENSE-JQUERY.txt
                    │   ├── jquery.js
                    │   └── jquery.min.js
                    └── xregexp
                        ├── LICENSE-XREGEXP.txt
                        ├── xregexp.js
                        └── xregexp.min.js

18 directories, 84 files
```
</p>
</details><br/>

Maybe we should ignore some stuff with `-I`, e.g. `tree -I 'static|__pycache__'`? Though that would need some explanation, too.

Reply:
well spotted.  `tree -L 3` also produces a reasonably sane amount of output.

```
$ tree -L 3
.
├── blog
│   ├── __init__.py
│   ├── __pycache__
│   │   ├── __init__.cpython-36.pyc
│   │   ├── [...]
│   ├── admin.py
│   ├── forms.py
│   ├── migrations
│   │   ├── 0001_initial.py
│   │   ├── __init__.py
│   │   └── __pycache__
│   ├── models.py
│   ├── static
│   │   └── css
│   ├── templates
│   │   └── blog
│   ├── tests.py
│   ├── urls.py
│   └── views.py
├── db.sqlite3
├── manage.py
├── mysite
│   ├── __init__.py
│   ├── __pycache__
│   │   ├── __init__.cpython-36.pyc
│   │   ├── [...]
│   ├── settings.py
│   ├── urls.py
│   └── wsgi.py
└── static
    ├── admin
    │   ├── css
    │   ├── fonts
    │   ├── img
    │   └── js
    └── css
        └── blog.css
```


I'm not sure we really want to dive into an explanation of `tree` arguments though?  How about just using `ls`:

```
$ ls
blog  db.sqlite3  manage.py  mysite  static
$ ls blog/
__init__.py  __pycache__  admin.py  forms.py  migrations  models.py  static
templates  tests.py  urls.py  views.py
```
that seems simpler, maybe just go with that for now?

Reply:
Yeah, good call. 👍

`ls` [has been introduced](https://tutorial.djangogirls.org/en/intro_to_command_line/#list-files-and-directories) (at least to UNIXoid users) in **Introduction to the command-line interface**, so let's stick to that.